src/pipeline.ml

```diff
@@ -190,7 +192,7 @@ module Make (OCurrent : S.OCURRENT) = struct
           |> or_die
         in
         let windows_port = match distro with  `Windows (port, _) -> Some port | _ -> None in
-        let repo_id = install_compiler ~os_family ~arch ~ocluster ~switch ~push_target ?windows_port opam_image in
+        let repo_id = install_compiler ~arch ~distro ~ocluster ~switch ~push_target ?windows_port opam_image in
```

Comment:
It is rather bike-sheddy, but the diff would be quieter and it's slightly more logical for `~distro` to be first, as `~os_family` was (here, and elsewhere)